app/auth/routes.py

```diff
@@ -59,6 +61,59 @@ def register():
             "msg": "Error in request method"
         }), 400
 
+@app.route('/login/', methods=['POST'])
+def login():
+    if current_user.is_authenticated:
+        return jsonify({
+                    'error': False,
+                    'code': 200,
+                    'title': 'User is authenticated',
+                    'msg': 'User is authenticated'
+                }),200
+    try:
+        data = request.get_json()
+    except Exception as e:
+        return jsonify({
+            'error': True,
+            'code': 400,
+            'title': 'Request failed',
+            'msg': str(e)
+            }), 400
+    user = User.query.filter_by(username=data['username']).first()
```

Comment:
if no data has no key username your code will fail. For example if someone call the endpoint without "username" as field your code will give him error.

An example for here:
```
username = data.get("username", None)
if not username:
   # return error
user = User.query.filter_by(username=data['username']).first()
```

Reply:
Ok, done.